packages/lib-classifier/src/plugins/tasks/TextTask/components/TextTask/TextTask.js

```diff
@@ -0,0 +1,75 @@
+import { PlainButton } from '@zooniverse/react-components'
+import { Box, FormField, TextArea } from 'grommet'
+import React from 'react'
+
+function TextTask (props) {
+  const { autoFocus, disabled, task } = props
+  const defaultValue = task.annotation ? task.annotation.value : ''
```

Comment:
Can we have a default value set in props and/or isn't there a default set in the store model?

Reply:
This is initially undefined because the annotation isn’t created until the first call to `addAnnotation()`. Using the model makes much more sense. #1228 tries to make tasks responsible for their default annotations.

Reply:
I added `task.defaultAnnotation` as a view in the model and return that if the stored annotation is undefined.